kids/lib/components/page/sections/poster_section.dart

```diff
@@ -23,91 +28,159 @@ class PosterSection extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final bp = ResponsiveBreakpoints.of(context);
     final double sectionSpacing = bp.smallerThan(TABLET) ? 20 : 28;
+
+    final posterItems = useMemoized(() {
+      return data.items.items.asMap().entries.map((kv) {
+        final item = kv.value;
+        return _createPosterItem(context, ref, item, kv.key);
+      }).toList();
+    }, [data.items.items]);
+
+    final notificationsEnabledSetting = ref.watch(settingsProvider.select((s) => s.notificationsEnabled));
+    final notificationPromptEnabled = ref.watch(featureFlagsProvider.select((flags) => flags.kidsNotificationPrompt));
+    final notificationPromptPosition = ref.watch(featureFlagsProvider.select((flags) => flags.kidsNotificationPromptPosition));
+    final showAfterDismissal = ref.watch(featureFlagsProvider.select((flags) => flags.kidsNotificationPromptAfterDismissal));
+    final showAfterDismissalCount = ref.watch(featureFlagsProvider.select((flags) => flags.kidsNotificationPromptAfterDismissalCount)) ?? 3;
+
+    final shouldShowPrompt = useState<bool>(false);
+
+    // Check if prompt should be shown on first render and when configuration changes
+    useEffect(() {
+      if (!notificationPromptEnabled || notificationsEnabledSetting == true || notificationPromptPosition == null) {
```

Comment:
```suggestion
      if (!notificationPromptEnabled || notificationsEnabledSetting || notificationPromptPosition == null) {
```


Probably

Reply:
the `notificationsEnabledSetting` is null by default, and will be set to either false or true when the user explicitly accepts or denies the permission request. So I thought I would be explicit here, but yeah probably unnecessary